orbit/pkg/execuser/execuser_darwin.go

```diff
@@ -41,8 +41,15 @@ func run(path string, opts eopts) (lastLogs string, err error) {
 		}
 	}
 
-	arg = append([]string{"-u", opts.user, "/usr/bin/open"}, arg...)
-	cmd := exec.Command("sudo", arg...)
+	var cmd *exec.Cmd
+	// If we have a user in the options, use sudo to run "open" as that user
+	if opts.user != "" {
+		arg = append([]string{"-u", opts.user, "/usr/bin/open"}, arg...)
+		cmd = exec.Command("sudo", arg...)
+	} else {
+		// Otherwise, just run "open" as the current user
+		cmd = exec.Command("/usr/bin/open", arg...)
```

Comment:
See https://github.com/fleetdm/fleet/commit/866d8bcc0073cf7837d60d43eec0b54ad718375f#diff-381da9d8ba1c754b060ea28c4cef30228716b958e066410842da3c85c8ec9a6eL44, this was the command we had before patching to use `sudo -u`